library/pp/mcall_spec.rb

```diff
@@ -0,0 +1,6 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'pp'
+
+describe "PP.mcall" do
```

Comment:
PP.mcall seems to be intended as private, so should not be spec'd, can you remove the file?